box2d3-wasm/README.md

```diff
@@ -1,20 +1,67 @@
-# Box2d3-wasm
+# box2d3-wasm
 
-Let's try and use box2d v3 on the web.
+Let's try and use [Box2D v3][] on the web via [wasm][]!
```

Comment:
the URLs(?) here look unfinished

Reply:
oh, I see, they get moved down to the bottom of the page…

Reply:
yeah some fancy markdown there haha, I could just link them up top if you want